### PR TITLE
fix: Make group exclusion work with null group id

### PR DESCRIPTION
**Pull request by fpacifici**
Right now group id exclusion breaks with issueless events because the query looks like this:

```
SELECT 
    event_id, 
    nullIf(group_id, 0) AS group_id, 
    project_id, 
    timestamp
FROM sentry_dist
PREWHERE event_id = 'af738635ed6649c4ad81a3bf0d492516'
WHERE 
(timestamp >= toDateTime('2019-08-15T21:53:15')) 
AND (timestamp < toDateTime('2019-08-20T21:53:15')) 
AND (deleted = 0) 
AND (project_id IN (11276)) 
AND (group_id NOT IN (1171086930))
```

if group_id is null nothing is returned. Adding assumeNotNull will fix it

### CODE REVIEW

File: snuba/api.py [hunk 1]
```diff
@@ -375,7 +375,7 @@ def parse_and_run_query(dataset, body, timer):
     table = dataset.get_schema().get_table_name()
 
     sql = format_query(dataset, body, table, prewhere_conditions, final)
-
+    print(sql)
```
Comment (dashed) [inline snuba/api.py]: Stray `print()` 👀 

(unless this is meant to be here)

**Review by dashed — COMMENTED** [review]
Looks good to me.

This pretty aligns with the conversations on Slack, this doc (https://clickhouse.yandex/docs/en/query_language/functions/functions_for_nulls/#assumenotnull), and the `conditions` param on the README. 

I'll let someone from @getsentry/sns to formally approve this PR.

-----

@fpacifici thank you for the fix! 👍

File: snuba/api.py (hunk 1 above)
Comment (fpacifici) [reply snuba/api.py]: oooops I forgot to commit before pushing  to remote

**Review by tkaemming — APPROVED** [review]
I think we'll need to find a way to apply this to any expression including `group_id` when dealing with issueless events, but this at least solves the immediate problem.

**Pull request merged**